src/main/scala/edu/ie3/mobsim/model/ChargingStation.scala

```diff
@@ -24,7 +24,7 @@ case class ChargingStation(
     private val evcsType: ChargingPointType,
     private val evcsLocationType: EvcsLocationType,
     private val chargingPoints: Int,
-    private var homeChargingStationAssignedToPOI: Boolean
+    private val homeChargingStationAssignedToPOI: Boolean
 ) {
```

Comment:
You could make all fields of `ChargingStation` public by removing the `private val` properties. Then, you could also remove the methods below